analyses/NC_cd_2010/02_setup_NC_cd_2010.R

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Set up redistricting simulation for `NC_cd_2010`
+# © ALARM Project, April 2022
+###############################################################################
+cli_process_start("Creating {.cls redist_map} object for {.pkg NC_cd_2010}")
+
+map <- redist_map(nc_shp, pop_tol = 0.005,
+    existing_plan = cd_2010, adj = nc_shp$adj)
+
+map <- map %>%
+    mutate(pseudo_county = pick_county_muni(map, counties = county, munis = muni,
+        pop_muni = get_target(map)))
```

Comment:
If we are not using pseudocounties in the simulation, we can remove `pseudo_county = pick_county_muni(map, counties = county, munis = muni, pop_muni = get_target(map))`